torchgeo/datasets/idtrees.py

```diff
@@ -459,7 +459,7 @@ def normalize(x: Tensor) -> Tensor:
         hsi = normalize(sample["hsi"][hsi_indices, :, :]).permute((1, 2, 0)).numpy()
         chm = normalize(sample["chm"]).permute((1, 2, 0)).numpy()
 
-        if "boxes" in sample:
+        if "boxes" in sample and len(sample["boxes"]):
```

Comment:
In the latest version of torchvision, `draw_bounding_boxes` now crashes if you give it an empty list of boxes